main.go

```diff
@@ -61,172 +79,216 @@ func init() {
 }
 
 func main() {
-	fmt.Printf("Starting estafette-gke-preemptible-killer (version=%v, branch=%v, revision=%v, buildDate=%v, goVersion=%v)\n",
-		version, branch, revision, buildDate, goVersion)
+	kingpin.Parse()
+
+	// log startup message
+	Logger.Info().
+		Str("branch", branch).
+		Str("revision", revision).
+		Str("buildDate", buildDate).
+		Str("goVersion", goVersion).
+		Msg("Starting estafette-gke-preemptible-killer...")
 
 	kubernetes, err := NewKubernetesClient(os.Getenv("KUBERNETES_SERVICE_HOST"), os.Getenv("KUBERNETES_SERVICE_PORT"),
-		os.Getenv("KUBERNETES_NAMESPACE"), os.Getenv("KUBECONFIG"))
+		os.Getenv("KUBERNETES_NAMESPACE"), *kubeConfigPath)
```

Comment:
I kept these environment variable there because it is specific to the Kubernetes client and hardcoded in the k8s library: https://github.com/ericchiang/k8s/blob/master/client.go#L252